client/js/lounge.js

```diff
@@ -275,7 +286,7 @@ $(function() {
 
 			chat.find(".chan.active .chat").trigger("msg.sticky"); // fix growing
 		})
-		.tab(completeNicks, {hint: false})
+		.tab(completeNicks(false), {hint: false})
```

Comment:
Come on, there is no unnecessary complexity in this whatsoever.

- @yashsriv's solution: a function that returns a function, called three times with `true`/`false` as an argument.
- Your solution: that inner function, called three times with `(word) => completeNicks(word, false/true)`

i.e. the exact same thing, just with the argument is shifted. At this point, that's bikeshedding.

> the way it's used does not warrant a "dynamic" function.

What does that even mean?